docs/devenv_cloud.md

```diff
@@ -0,0 +1,159 @@
+# MicroShift Development Environment in Cloud
+
+## Introduction
+
+The MicroShift development environment should be set up on a **local workstation** as a collection of virtual machines.
+* The [development virtual machine](./devenv_rhel8.md#create-development-virtual-machine) used for compiling and running the MicroShift binaries and building [RHEL for Edge Installer ISO](./rhel4edge_iso.md#build-rhel-for-edge-installer-iso) images
+* The [RHEL for Edge virtual machine](./rhel4edge_iso.md#install-microshift-for-edge) used for running MicroShift in a clean, production-like environment
```

Comment:
Not for this PR, but we'll eventually need to remove "RHEL for Edge" everywhere, as this doens't exist / brand discourages its use. We'll probably want to replace with "Red Hat Device Edge".

Reply:
Not sure. I'm still seeing RHEL for Edge. For instance [here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/composing_installing_and_managing_rhel_for_edge_images/managing-rhel-for-edge-images_composing-installing-managing-rhel-for-edge-images)

Reply:
Yeah. That'll change eventually. Anyway, not relevant to this PR as I said.